homework.md

```diff
@@ -39,75 +43,91 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 }
 ```
+h1 {
+
+}
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+It's a language used to make a website become interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
+What three problems does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. Tracks changes 2.Instant feedback on collaborative work. 3.Version control
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
 You delete it
+Fork is another term for 'copy'.
```

Comment:
Hi Elisha,

The answer is partially correct. Can you expand on your answer and describe the process.

All the best,

Aaron